literature.bib

```diff
@@ -12893,6 +13261,29 @@ @inproceedings{M02
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2001 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{KKT:ASE11,
+	author = {Timo {Kehrer} and Udo {Kelter} and Gabriele {Taentzer}},
+	booktitle = ASE,
+	publisher = ACM,
+	title = {{A Rule-Based Approach to the Semantic Lifting of Model Differences in the Context of Model Versioning}},
+	year = 2011,
+	pb-tags = {semantic lifting},
+	pages = {163.-172}
+}
+
+@inproceedings{GYK:ICSE01,
+	author = {Griswold, William G. and Yuan, Jimmy J. and Kato, Yoshikiyo},
+	title = {{Exploiting the Map Metaphor in a Tool for Software Evolution}},
+	year = 2001,
+	isbn = {0769510507},
+	publisher = IEEE,
+	address = {USA},
```

Comment:
?

Reply:
That is the most precise address I could find for IEEE for ICSE 2001. Should I remove it?

Reply:
Other ICSE events with publisher IEEE have address Washington (see BM:ICSE98)

Reply:
I see. I was unsure if addresses of publishers might change over the years.

Reply:
I set this now to washington.